README.md

```diff
@@ -1,97 +1,111 @@
+NESTML is a domain-specific language that supports the specification of neuron models in a precise and concise syntax, based on the syntax of Python. Model equations can either be given as a simple string of mathematical notation or as an algorithm written in the built-in procedural language. The equations are analyzed by the associated toolchain, written in Python, to compute an exact solution if possible or use an appropriate numeric solver otherwise.
 
 ## Directory structure
 
-`models` - Example neuron models in NestML format.
+`models` - Example neuron models in NESTML format.
 
-`pynestml` - The source code of PyNestML.
+`pynestml` - The source code of the PyNESTML toolchain.
 
 `tests` - A collection of tests for testing of the toolchain's behavior.
 
-`doc` - The documentation of the modeling language NestML as well as processing toolchain PyNestML.
+`doc` - The documentation of the modeling language NESTML as well as processing toolchain PyNESTML.
 
 `extras` - Miscellaneous development tools, editor syntax highlighting rules, etc.
 
 ## Installing NESTML
 
-### Requirements
+Please note that only Python 3 is supported. The instructions below assume that `python` is aliased to or refers to `python3`, and `pip` to `pip3`.
 
-In order to execute the language tool-chain, Python in version 2 or 3 is required.
-To ensure correct installation and resolving of dependencies, Python's package manager [pip](https://pip.pypa.io/en/stable/installing/), the distribution tool [setuptools](https://packaging.python.org/tutorials/installing-packages/) as well as the python headers (``python-dev`` on Ubuntu/Debian, ``python-devel`` on most other linux platforms) package are required and should be installed in advance. The setup file additionally installs the following components:
+### Installing the latest release from PyPI
 
-* [SymPy in the version >= 1.1.1](http://www.sympy.org/en/index.html)
-* [NumPy in the version >=1.8.2](http://www.numpy.org/)
-* [Antlr4 runtime environment in the version >= 4.7](https://github.com/antlr/antlr4/blob/master/doc/python-target.md)
+The easiest way to install NESTML is to use the [Python Package Index (PyPI)](https://pypi.org). This requires the Python package management system `pip` to be installed. In Ubuntu, Mint and Debian Linux you can install `pip` as follows:
 
-In the case that no 'enum' package is found, additionally, enum34 has to be updated by
 ```
-pip install --upgrade pip enum34
+sudo apt install python3-pip
 ```
 
-First of all, you should make sure that all required packages are installed.
-Requirements are stored in ``requirements.txt`` and can be installed in one step through pip by running
+NESTML can then be installed into your local user directory via:
+
 ```
-pip install -r requirements.txt
+pip install nestml --user
+```
+
+Optionally, you can run `pip` as root to install NESTML system-wide:
+
 ```
+sudo pip -H install nestml
```

Comment:
would suggest to write `sudo -H pip install nestml` since `-H` is a flag to sudo, not pip.

Reply:
It looks like running `pip` as root is actively discouraged, because of obvious security risks: anyone can upload malicious code to PyPI, which will then (in `setup.py`) be run as root. Shall I remove the suggestion to `sudo pip`?

Reply:
Yes, remove it. I guess that people will find out anyway if they need to.